services/job_runner_test.go

```diff
@@ -37,18 +40,17 @@ func TestJobRunner_ExecuteRun(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 
 			var run models.JobRun
-			mockServer, cleanup := cltest.NewHTTPMockServer(t, 200, "POST", test.runResult,
+			mockServer, _ := cltest.NewHTTPMockServer(t, 200, "POST", test.runResult,
```

Comment:
Is there a reason we don’t cleanup the mock server?

Reply:
It's a misnamed variable. It doesn't actually return a cleanup method. It returns a method with an assertion to ensure it was called.

In this particular test there will be 1 example that does get called and one that doesn't.

https://github.com/smartcontractkit/chainlink/blob/master/internal/cltest/mocks.go#L355